### PR TITLE
Adding additional Docker log drivers.

### PR DESCRIPTION
**Pull request by justnom**
Adding additional `log_driver` choices:
- journald
- gelf
- fluentd

Compatible with Docker version >= 1.8.0.

### CODE REVIEW

File: cloud/docker/docker.py [hunk 1]
```diff
@@ -97,9 +97,12 @@
       - You can specify a different logging driver for the container than for the daemon.
```
Comment (justnom) [inline cloud/docker/docker.py:97]: This description is a little verbose - shouldn't it just link to the [Docker reference](https://docs.docker.com/reference/logging/overview/) page instead?

**Comment by gregdek** [comment]
@cove @joshuaconner @softzilla @smashwilson Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

**Comment by zentavr** [comment]
It would be very great to have this with Ansible 2.0.0 :)

**Comment by zentavr** [comment]
BTW, seems like very soon there will be AWS Cloudwatch driver.
https://github.com/docker/docker/pull/15495/files

**Comment by hourliert** [comment]
:+1:

**Pull request merged**